src/components/moderation/moderation-common.ts

```diff
@@ -647,32 +652,7 @@ export abstract class ModerationComponent extends BotComponent {
     ) {
         try {
             for (const user of users) {
-                if (this.wheatley.is_authorized_mod(user)) {
-                    await this.reply_with_error(command, moderation_on_team_member_message);
-                    continue;
-                }
-                const base_moderation: basic_moderation_with_user = { ...basic_moderation_info, user: user.id };
-                if (!this.is_once_off && (await this.is_moderation_applied(base_moderation))) {
-                    await this.reply_with_error(command, `${user.displayName} is already ${this.past_participle}`);
-                    continue;
-                }
-                const moderation: moderation_entry = {
-                    ...basic_moderation_info,
-                    case_number: -1,
-                    user: user.id,
-                    user_name: user.displayName,
-                    moderator: command.user.id,
-                    moderator_name: (await command.get_member()).displayName,
-                    reason,
-                    issued_at: Date.now(),
-                    duration: parse_nullable_duration(duration_string),
-                    active: !this.is_once_off,
-                    removed: null,
-                    expunged: null,
-                    link: command.get_or_forge_url(),
-                };
-                await this.notify_user(user, this.past_participle, moderation);
-                await this.issue_moderation(moderation);
+                await this.moderation_issue_handler(command, user, duration_string, reason, basic_moderation_info);
```

Comment:
Can we actually revert this change for this PR? The problem is I think a lot has to be updated `moderation_issue_handler` (and anything called by it) so that all replies will either be replies or followups as appropriate

Reply:
I guess we can, I just noticed that it's duplicating some logic but not all the logic, and missing some of the checks we have in the other handler.